src/Simple.OData.Client.UnitTests/FluentApi/FindTypedTests.cs

```diff
@@ -287,6 +287,17 @@ public async Task Get()
             Assert.Equal(1, category.CategoryID);
         }
 
+        [Fact]
+        public async Task GetWithAltKey()
```

Comment:
A test which uses the `IDictionary<string, object>` overload should be included as well.

Reply:
I refactored the object overload to call the dictionary overload underneath so the test covers both cases. Let me know if still want a seperate test.